Assets/Scripts/PAXSpawner/PassengerSpawner.cs

```diff
@@ -0,0 +1,36 @@
+using Features.Managers;
+using System.Collections;
+using UnityEngine;
+
+public class PassengerSpawner : MonoBehaviour
+{
+    [SerializeField] private bool _canSpawnPassengerOnTime;
```

Comment:
Not clear what this variable does. Either improve the readability or potentially add a Tooltip